source/index.md

```diff
@@ -600,9 +602,46 @@ object][ApplePay-PaymentToken] for more information.
   subsequent recurring authorization.
 </p>
 
-##### Method: `mobilepayonline`
 
-<dl class="dl-horizontal">
+---
+##### **Method**: `googlepay`
+
+To accept a payment using Google Pay, apart from the complete payment method
+token and merchant ID, the derived shared secret is required. This must be
+derived according to the documentation.
+
+The cryptography involved is described [here][GooglePay-PaymentCryptography].
+
+<dl>
+  <dt>googlepay[token]</dt>
+  <dd>[:json:] <br /> Raw payment method token as received in response from Google. UTF-8
+    encoded serialization of a JSON dictionary.</dd>
+  <dt>googlepay[shared_secret]</dt>
+  <dd>
+    The shared secret derived from the ephemeral public key and your private key, Base64 encoded.
+  </dd>
+  <dt>googlepay[merchant_id]</dt>
+  <dd>
```

Comment:
```suggestion
  <dd>`merchant:[:hex:]`
```

But then we should likely have all the regexp-ish format descriptions be `code-formatted` ... dunno how to handle this. Perhaps the textual version below is better, and then just have `<dd>[\x20-\x7E]{10,??}`. WDYT?

Are there any length requirements for the merchant ID btw.?

Reply:
I cannot find any information about the format of merchantID at all, sadly.

Hmm. I don't know. I'll have to try a few things.

Reply:
The most valid thing would be `merchant:.+`, but it doesn't look that good rendered.

Reply:
I think I'm renaming it, it's better referred to as `recipient_id`. Which is `merchant:<merchant_id>`.

Reply:
I'm somewhat a fan of your renaming suggestion — we already have quite some "things" named "merchant id" 🙁